pages/common/vue-init.md

```diff
@@ -0,0 +1,18 @@
+# vue-init
+
+> Legacy init command.
+> Require `vue-cli@2` or `@vue/cli` with `@vue/cli-init`.
```

Comment:
I'm not sure we need this line. The point of tldr pages is to show how to _use_ a command, not to install it. I think we can assume that if the user is looking up the page, then they already have it installed or know where to find that information :-)

Reply:
Applied changes to this.